app/resources/admin.html.template

```diff
@@ -666,6 +666,26 @@
   </fieldset>
 
   <fieldset>
+    <legend>Notification settings</legend>
+
+    <div class="config">
+        <label for="admin_user_email">Default admin user email:</label>
```

Comment:
You should rather explain that this is the email address which we send notification emails to. Otherwise it's not clear what is "admin user email".